# Pell_GBS_Genomics:
This repository contains scripts used to collect and analyze data from whole genome sequences of clinical _Streptococcus agalactiae_ isolates for the following manuscript. 

## Genomic adaptation in group B _Streptococcus_ following intrapartum antibiotic prophylaxis and childbirth.
### *Macy E. Pell, Heather M. Blankenship, Jennifer A. Gaddy, H. Dele Davies, Shannon D. Manning*

### Citation
_Peer-reviewed citation pending_

[View the preprint here!](https://www.biorxiv.org/content/10.1101/2024.04.01.587590v1)
Pell, M.E.; Blankenship, H.M.; Gaddy, J.A.; Davies, H.D.; Manning, S.D. Intrapartum Antibiotic Prophylaxis Selects for Mutators in Group B Streptococci among Persistently Colonized Patients. bioRxiv 2024, 2024.04.01.587590, doi:10.1101/2024.04.01.587590.

### Abstract

Through vaginal colonization, GBS causes severe outcomes including neonatal sepsis and meningitis. Although intrapartum antibiotic prophylaxis (IAP) has reduced neonatal disease rates, GBS can persist in the genitourinary tract even after antibiotic administration. To determine if IAP selects for genomic signatures that enhance GBS survival and persistence, we compared the isolates from individuals before (prenatal) and after (postpartum) IAP/childbirth. Among 34 of the paired strains from participants with persistent colonization, 31 (91.2%) clustered together in a core gene phylogeny, suggesting colonization with highly similar strains before and after IAP. A core-gene mutation analysis, however, identified mutations in 74% (n=23) of these 31 postpartum genomes when each genome was compared to its respective prenatal genome from the same individuals. Several strains acquired mutations in the same genes, though two postpartum strains accounted for most of the mutations. These two strains were classified as mutators based on high mutation rates and mutations within DNA repair system genes. Changes in biofilm production were observed in a subset of postpartum strains, which is supported by the presence of point mutations in genes linked to survival and colonization. These findings suggest that exposures encountered during pregnancy and childbirth may select for mutations and phenotypes that promote adaptation and survival in vivo. Enhanced survival in the genitourinary tract can lead to persistent colonization, increasing the likelihood of invasive disease in subsequent pregnancies and in newborns (late-onset infections) following IAP cessation.

### Importance

GBS remains a major cause of neonatal sepsis, pneumonia, and meningitis despite the common use of IAP that aims to eradicate maternal colonization, the main risk factor for neonatal disease. Although IAP has reduced the incidence of early-onset neonatal infections, it has had no impact on late-onset infections in babies between 7 days and 3 months of age. Since colonization is intermittent and GBS has been shown to persist in the genitourinary tract despite antibiotic exposure, more research is needed to understand mechanisms of adaptation. By comparing the genomes of GBS strains recovered before (prenatal) and after (postpartum) IAP and childbirth, this study demonstrates how selective pressures shape GBS evolution, favoring traits that promote survival and persistence. Understanding adaptative traits is essential for improving diagnostics, refining prophylaxis strategies, and guiding development of more effective prevention practices that can reduce the likelihood of GBS transmission to neonates.

### Data Availability
The raw sequencing reads for all genomes examined in this study are available on NCBI GenBank under various BioProjects listed below:
* PRJNA1070672 (not yet publicly available), [PRJNA161909](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA161909), [PRJNA86455](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86455), [PRJNA86457](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86457), [PRJNA86459](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86459), [PRJNA86461](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86461), [PRJNA86463](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86463), [PRJNA86465](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86465), [PRJNA86467](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86467), [PRJNA86469](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86469), [PRJNA86471](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86471), [PRJNA86473](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86473), [PRJNA86475](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86475), [PRJNA86477](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86477), [PRJNA86479](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86479), [PRJNA86481](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86481), [PRJNA86483](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86483), [PRJNA86485](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86485), [PRJNA86487](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86487), [PRJNA86489](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86489), [PRJNA86491](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86491), [PRJNA86493](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86493), [PRJNA86495](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86495), [PRJNA86497](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86497), [PRJNA86499](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86499), [PRJNA86501](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86501), [PRJNA86503](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86503), [PRJNA86505](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86505), [PRJNA86507](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86507), [PRJNA86509](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86509), [PRJNA86511](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86511), [PRJNA86513](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86513), [PRJNA86515](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86515), [PRJNA86517](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86517), [PRJNA86519](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86519), [PRJNA86521](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86521), [PRJNA86523](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86523), [PRJNA86525](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86525), [PRJNA86527](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86527), [PRJNA86529](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86529), [PRJNA86531](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86531), and [PRJNA86533](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA86533)
* All BioProject, BioSample, and Genome ID information along with isolate and sequencing characteristics are listed in supplementary tables 1 and 2 associated with the manuscript

### Acknowledgements and Funding
We thank the Research Technology Support Facility and Michigan Department of Agriculture and Rural Development for DNA sequencing, as well as the National Center for Streptococcus (Edmonton, Alberta), particularly Marguerite Lovgren, for strain collection and serotyping. Financial support was provided by the Michigan State University (MSU) Research Foundation (to SDM) and in part, by the Michigan Sequencing and Academic Partnerships for Public Health Innovation and Response (MI-SAPPHIRE) initiative at the MDHHS via CDC through the Epidemiology and Laboratory Capacity for Prevention and Control of Emerging Infectious Diseases Enhancing Detection Expansion program (6NU50CK000510-02-07; to HMB and SDM). Additional support was provided by the National Institutes of Health (R01HD090061 to JAG and SDM, R01AI134036 to JAG) and Department of Veterans Affairs Merit Award I01BX005352 (Office of Research) to JAG. Student support was provided to MEP by the MSU Department of MGI via the Philipp and Vera Gerhardt Travel and Ralph Evans Awards and the MSU College of Natural Science via two Outstanding Scholar Fellowships.

